src/layout/random.c

```diff
@@ -61,7 +61,7 @@ int igraph_layout_random(const igraph_t *graph, igraph_matrix_t *res) {
 
 /**
  * \function igraph_layout_random_3d
- * \brief Random layout in 3D
```

Comment:
I'd say if the brief description of `igraph_layout_random` is "Places the vertices uniform randomly on a plane.", then let's use a similar wording here:

"Places the vertices uniform randomly in a cube."

We can then add details about the coordinates of the corners of the cube in the main docstring but it is an unnecessary detail in a brief description (which appears in many places: section titles, TOC entries of the documentation etc).